src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_mixins.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ Allow rem units with px fallback
+*/
+
+$rem-baseline: $font-size-normal !default;
+$rem: true !default;
+$rem-px-only: false !default;
+
+// list-separator polyfill by Hugo Giraudel (https://sass-compatibility.github.io/#list_separator_function)
+@function rem-separator($list) {
+    @if function-exists("list-separator") == true {
+        @return list-separator($list);
+    }
+
+    $test-list: ();
+    @each $item in $list {
+        $test-list: append($test-list, $item, space);
+    }
+
+    @return if($test-list == $list, space, comma);
+}
```

Comment:
I think we do not need this polyfill because we already use ruby sass 3.4 and have no intention of downgrading.